client/src/services/userService.js

```diff
@@ -8,6 +8,22 @@ export default {
                 passwort: passwort
             }
         });
+        return res.data || [];
+    },
+    submitUser: async (email, passwort) => {
+        let res = await axios.post('/api/user', {
+            email: email,
+            passwort: passwort,
+
+        })
+
         return res.data || [];
     }
-}
\ No newline at end of file
+
+
+}
+
+export function  isAuth(){
+    return localStorage.getItem('email');
+};
```

Comment:
this method checks if user is already authenticated ...I'm using it for links protection. 
it means if user has this email in local Storage then auth=success and you can see all links . if not ...nothing happens